torchmetrics/functional/text/bert.py

```diff
@@ -376,7 +376,7 @@ def _get_precision_recall_f1(
 
 
 def _get_hash(model_name_or_path: Optional[str] = None, num_layers: Optional[int] = None, idf: bool = False) -> str:
-    """Copied from https://github.com/Tiiiger/bert_score/blob/master/bert_score/utils.py and adjusted."""
+    r"""BERT_score\_"""
```

Comment:
```suggestion
    r"""`BERT score`_"""
```

Reply:
My bad, I will make it correct.

Reply:
Besides this, If anything seems missing or needs to be corrected. Let me know

Reply:
In Your suggestion, you want r"""`BERT score`_"""


My question is in the link.rst this is mentioned as BERT_score, So Do I need to remove underscore(_) while replacing it in Codebase or write it as it is written in links.rst